src/main-process/proxy/test-actions/transform.ts

```diff
@@ -81,7 +81,14 @@ export const getTransformResult = async (
 };
 
 export type Extraction = { [parameter: string]: string | object };
-export type LoadmillRequest = { extract: Extraction[]; id: string; irrelevant?: boolean; method: string; };
+export type LoadmillRequest = {
+  description: string;
```

Comment:
can `description` be `undefined`?

Reply:
Yes. Although this is a rare case, we will handle it.